tutorials/data-lake-file-containers-hdlfscli/data-lake-file-containers-hdlfscli.md

```diff
@@ -173,29 +179,39 @@ Now click save at the top of the page.
 
 Next, we will verify that the configuration we did in Steps 5 & 6 work.
 
-The `<REST API Endpoint>` and `<Instance ID>` can be found in the SAP HANA Cloud Central Cockpit. `<PATH>` is the path to the corresponding certificate. Expect nothing to be returned by the below command.
+The `<REST API Endpoint>` and `<Instance ID>` can be found in the SAP HANA Cloud Central Cockpit. `<PATH>` is the path to the corresponding certificate.
+The following command wont return anything in the output.
 
-`hdlfscli -cert <PATH>/client.crt -key <PATH>/client.key -cacert <PATH>/ca.crt -k -s https://<REST API Endpoint> -filecontainer <Instance ID> ls`
+`hdlfscli -cert <PATH>\client.crt -key <PATH>\client.key -cacert <PATH>\ca.crt -k -s https://<REST API Endpoint> -filecontainer <Instance ID> ls`
 
-**[Optional]**: Configure a configuration file to make using the CLI simpler.
+**[Optional]:** Configure a configuration file to make using the CLI simpler.
 
-**Note**: The configuration will be placed in the user's root directory. It is saved as a JSON file that can be modified in any text editor.
+**Note:** The configuration will be placed in the **user's root directory**. It is saved as a JSON file that can be modified in any text editor.
```

Comment:
Please don't start a note with the label "Note". The formatting already makes it a note.
Please check the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9#mcetoc_1dtqfqchq21) for details.